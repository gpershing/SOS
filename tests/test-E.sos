x: float = 2.0E2
print(x)
