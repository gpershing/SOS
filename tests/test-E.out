2.0E2
